src/SARProcessing.jl

```diff
@@ -1,5 +1,7 @@
 module SARProcessing
+import ArchGDAL
```

Comment:
Are you extending the functionality of ArchGDAl? If not, it would be good practice to do using instead of import. If you want to only get the module name into scope you can do 'using ArchGDAL: ArchGDAL'

Reply:
We use ArchGDAL to load the Sentinel tiff files. It is the only package I have found so far that works with complex tiff and do not truncate the pixel values